src/library-authoring/course-import/CourseImportPage.jsx

```diff
@@ -95,19 +97,17 @@ export const CourseImportList = ({
       {
         courseCount > 0
           ? (
-            <ul className="library-list importable-course-list">
+            <>
               {courses.map((course) => (
-                <li key={course.id} className="library-item no-hover-bg">
```

Comment:
If you look at the console, you'll see React complaining about the missing `key={}`.  I think we need to add it back in.  (Like in the AuthoringPage.)

Reply:
fixed by https://github.com/brian-smith-tcril/frontend-app-library-authoring/commit/ff8d45a0a9f637a65b7d59f6b3d72330abbb302c